cmd/up/controlplane/list.go

```diff
@@ -52,6 +52,11 @@ func (c *listCmd) Run(p pterm.TextPrinter, pt *pterm.TablePrinter, cc *cp.Client
 		p.Printfln("No control planes found in %s", upCtx.Account)
 		return nil
 	}
+	return PrintControlPlanes(cpList, pt)
+}
+
+// Prints a list of control planes. This is also used by the get command
+func PrintControlPlanes(cpList *cp.ControlPlaneListResponse, pt *pterm.TablePrinter) error {
```

Comment:
We can avoid exporting here since it isn't used outside of this package 👍🏻 
```suggestion
	return printControlPlanes(cpList, pt)
}

// Prints a list of control planes. This is also used by the get command
func printControlPlanes(cpList *cp.ControlPlaneListResponse, pt *pterm.TablePrinter) error {
```